python/cuml/ts/stationarity.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+
+import statsmodels.tsa.stattools as st
+import pmdarima.arima as am
+
+from sklearn.linear_model import LinearRegression
+
+from IPython.core.debugger import set_trace
+
+def is_stationary(yi: np.ndarray, pval_threshold=0.05) -> bool:
```

Comment:
With the changes to use GPU memory overall, this could be using cupy, numba kernels or C++ cuda kernels probably